utils/plotting.py

```diff
@@ -129,32 +129,32 @@ def new_show(t, n=10, width=140, truncate=40, types=True):
              'eas': 'East Asian',
              'fin': 'Finnish',
              'eur': 'European',
-             'nfe': 'European',
+             'nfe': 'Non-Finnish European',
              'sas': 'South Asian',
              'mde': 'Middle Eastern',
              'asj': 'Ashkenazi Jewish',
              'uniform': 'Uniform',
              'sas_non_consang': 'South Asian (F < 0.05)',
              'consanguineous': 'South Asian (F > 0.05)',
              'exac': 'ExAC',
-             'bg': 'Bulgaria (Eastern Europe)',
-             'de': 'Germany',
-             'ee': 'Estonia',
-             'es': 'Spain',
-             'fi': 'Finland',
-             'gb': 'UK',
+             'bgr': 'Bulgarian',
```

Comment:
We had semi-intentionally had the parenthetical here (since we're not necessarily convinced this is a strictly-Bulgarian signature, but might be picking up any Eastern European)

Reply:
So want it back?

Reply:
Yes I think so